sdk/graphql-transport/README.md

```diff
@@ -68,7 +68,7 @@ If an unsupported parameter is used, the request will error, or fallback to JSON
   - missing `id` for `events`
 - `getStakes` and `getStakesByIds`
   - missing `validatorAddress`
-- `getLatestIotaSystemState`
+- `getSupportedIotaSystemState`
```

Comment:
would you remove `getLatestIotaSystemState` and replace it by this? maybe we could keep both, and actually i think we might be missing here the v2 version?